src/Crowdfunding/N3O.Umbraco.Crowdfunding/CrowdfundingConstants.cs

```diff
@@ -67,10 +70,10 @@ public static class Properties {
             public const string Title = "title";
             public const string Type = "type";
             public const string Amount = "amount";
-            public const string FundDimension1 = "fundDimension1";
-            public const string FundDimension2 = "fundDimension2";
-            public const string FundDimension3 = "fundDimension3";
-            public const string FundDimension4 = "fundDimension4";
+            public const string Dimension1 = "dimension1";
```

Comment:
Leave these as fundDImension please as gets confusing when we introduce attribution (which we will later) which also has dimensions.

Reply:
Have changed these back but FundraiserAllocationElement was inhertiting from IFundDimensionValues which has these as "Dimension1", "Dimension2" etc, so removed that bcz didn't want to change that as it ws being used in other places as well Or would it be better to just add [UmbracoProperty("alias")] attribute?